lib/4.06.1/bsdep.ml

```diff
@@ -45742,6 +45744,7 @@ let app_exp_mapper
                    pexp_attributes }
          else   {e with pexp_attributes } (* BS_NATIVE branch*)
      )
+
```

Comment:
Extra newline

Reply:
this is generated code, but it comes because source file had no newline at the end of file